### DISCUSSION TITLE
Resolution Issue

### DISCUSSION THREAD

**Discussion by Humphhh** [PlayCover/PlayCover#324] [General]
Recently updated to this community fork from the pink-icon original. Nice work guys.
Only problem I met is that in Genshin Impact, the iPad Model settings and resolution settings are not in effect. Though I selected the 5th gen settings, the game still thinks that its is running on an iPad Pro 3rd gen. The resolution and settings are not from 5th gen, and it's not as clear as the Genshin running on a 4th gen iPad Pro, which is similar to the 3rd gen.
What I have is a 16GB version M1 MBP, with macOS 12.3.1.

**Comment by ZhichGaming** [comment]
This issue is solved in the nightlies.

**Reply by Humphhh** [reply]
thx!

**Reply by ZhichGaming** [reply]
Does it work? If yes, then mark the comment as the answer please.

**Reply by Humphhh** [reply]
I have the [nightly release #6](https://github.com/PlayCover/PlayCover/actions/runs/3017810635)0 downloaded, however the picture quality remains the same.

**Reply by ZhichGaming** [reply]
> I have the [nightly release #6](https://github.com/PlayCover/PlayCover/actions/runs/3017810635)0 downloaded, however the picture quality remains the same.

Did you try changing in-game resolution? Are you using m1 model identifier?

**Reply by Humphhh** [reply]
I have tried and I'm using the M1 identifier. Game UI is in lower resolution notably compared to the iPad. Gaming is the same.
![Uploading 截屏2022-09-09 22.55.59.png…]()
Screenshot (iPad Pro 4th Gen) on the left and M1 MBP is the right one.
